app/controllers/passengers_controller.rb

```diff
@@ -48,8 +48,11 @@ def destroy
   private
 
   def passenger_params
-    params.require(:passenger).permit(:name, :address, :email, :phone,
-      :wheelchair, :active, :permanent, :expiration, :note)
+    permitted_params = params.require(:passenger).permit(:name, :address, :email, :phone,
+        :wheelchair, :active, :permanent, :expiration, :note)
+    unless @current_user.admin?
+      permitted_params = permitted_params.except(:active, :permanent)
+    end
```

Comment:
@Anbranin not sure what you mean. But currently what I see on GH is that this method still ends with the output of a conditional, as @werebus originally mentioned.